src/core/sys/posix/sys/ioctl.d

```diff
@@ -323,13 +323,7 @@ else version (OSX)
     import core.sys.posix.termios; // termios
     import core.sys.posix.sys.time; // timeval
 
-    struct winsize
```

Comment:
Not sure what you mean -- I only moved the 'struct winsize' from the version (OSX) block into ttycom.d. This is to reflect how it is placed in the C header files on OSX so someone moving to D from C on OSX knows where to look for this based on C documentation.